generator/osm2meta.cpp

```diff
@@ -240,11 +239,7 @@ std::string MetadataTagProcessorImpl::ValidateAndFormat_level(std::string v)
 {
   // Some mappers use full width unicode digits. We can handle that.
   strings::NormalizeDigits(v);
-  double levels;
```

Comment:
maybe add a comment that multiple-number / free-form values are allowed?

Reply:
:+1: Added comments for `building_levels` and `level`